common/src/main/java/com/microsoft/identity/common/internal/net/UrlConnectionHttpClient.java

```diff
@@ -44,6 +44,7 @@
  * <p>
  * Currently served as an adapter of {@link com.microsoft.identity.common.java.net.UrlConnectionHttpClient}
  */
+@edu.umd.cs.findbugs.annotations.SuppressFBWarnings("NM_SAME_SIMPLE_NAME_AS_SUPERCLASS")
```

Comment:
```suggestion
@edu.umd.cs.findbugs.annotations.SuppressFBWarnings("NM_SAME_SIMPLE_NAME_AS_SUPERCLASS",
         justification = "This class kept its original name to avoid breaking change during the refactoring process." +
                 "Once the process is done, this class will be removed entirely. ")
```

Reply:
already fixed in the commit above.